packages/next/client/image.tsx

```diff
@@ -45,6 +45,8 @@ const VALID_LAYOUT_VALUES = [
 ] as const
 type LayoutValue = typeof VALID_LAYOUT_VALUES[number]
 
+type PlaceholderValue = 'blur' | 'empty'
```

Comment:
@timneutkens @Timer Do you think we should use `empty` or `none`? Do we have any precedence for this?

Reply:
I don't think `none` would be a good name since it may imply that we don't placeholder the image in any way, causing layout shift. `empty` was used in an effort to promote the understanding that there will at least be empty space.

Reply:
Another option here is `blank`. I agree that `none` might imply that we don't have a placeholder here.